ecs/daemon/outputs/cloudwatch_logs/task.conf

```diff
@@ -0,0 +1,30 @@
+
+[FILTER]
+    Name              ecs
+    Match             task.var.lib.docker.containers.*
+    ECS_Tag_Prefix    task.var.lib.docker.containers.
+    ADD               container    $ECSContainerName
+    ADD               ecs_task_id  $TaskID
+    ADD               ecs_family   $TaskDefinitionFamily
+
+[FILTER]
+    Name               aws
+    Match              task.var.lib.docker.containers.*
+    imds_version       v1
```

Comment:
In all config files, IMDS version used by AWS filters should be configured by the same single env var setting which defaults to v1.

Reply:
Actually, since we use host mode, its always safe to use V2 with default hop limit of one, so will just use that. No need to worry about this. V2 is always enabled by default now.